app/src/main/java/ch/epfl/sdp/ui/main/swipe/EventDetailFragment.java

```diff
@@ -4,25 +4,30 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.SeekBar;
-import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 import com.bumptech.glide.Glide;
-import com.google.firebase.firestore.GeoPoint;
-
 
 import ch.epfl.sdp.Event;
-import ch.epfl.sdp.R;
 import ch.epfl.sdp.databinding.FragmentDefaultEventBinding;
 
+/**
+ * Fragment to display details about an event.
+ * To be replaced !!!
```

Comment:
make it @Deprecated to discourage from using it until removed

Reply:
ça risque de faire un warning après